stress.yaml

```diff
@@ -27,19 +27,32 @@
     - set_fact: index={{ec2_tag_index}}
       when: index is not defined
 
+    - set_fact: taskset="taskset -c {{load_cpus}}"
+      when: load_cpus is defined
```

Comment:
@glommer are you using different load_cpus in each run or is it a property of the loader server?